101-vm-ws2016-visualstudio2019/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "itemDisplayName": "Deploy a Windows SV VM with Visual Studio.",
+  "description": "This template deploys a Windows Server VM with Visual Code Studio Community 2019, with a few options of VM. For that you need to inform the name of VM, admin user and admin password.",
```

Comment:
This template deploys a Windows Server VM with Visual Code Studio Community 2019, with a few options for the VM.  You can provide the name of VM, the admin username and admin password.